tests/callistocore/test_delivery.py

```diff
@@ -19,6 +19,275 @@
+        rendered_text = pdf_reader.getPage(1).extractText()
+        self.assertIn('Formset 1', rendered_text)
+        self.assertIn('Date text question?', rendered_text)
+        self.assertIn('4/16/2016', rendered_text)
+        self.assertIn('Formset 2', rendered_text)
+        self.assertIn('Multiline text question?', rendered_text)
+        self.assertIn('Multiline\ntext\nanswer', rendered_text)
+
+    def test_multiline_text_rendered(self):
+        multiline_question = '''[
+        { "answer": "Multiline\\ntext\\nanswer",
+        "id": 1,
+        "section": 1,
+        "question_text": "Multiline text question?",
+        "type": "MultiLineText"
+        }
+        ]'''
+        report = PDFFullReport(self.report, multiline_question)
+        output = report.generate_pdf_report(recipient=None, report_id=None)
+        rendered_report = BytesIO(output)
+        pdf_reader = PyPDF2.PdfFileReader(rendered_report)
+        rendered_text = pdf_reader.getPage(1).extractText()
+
+        self.assertIn('Multiline text question?', rendered_text)
+        self.assertIn('Multiline\ntext\nanswer', rendered_text)
+
+    def test_multiline_text_no_answer_rendered(self):
+        multiline_question = '''[
+        { "answer": "",
+        "id": 1,
+        "section": 1,
+        "question_text": "Multiline text question?",
+        "type": "MultiLineText"
+        }
+        ]'''
+        report = PDFFullReport(self.report, multiline_question)
+        output = report.generate_pdf_report(recipient=None, report_id=None)
+        rendered_report = BytesIO(output)
+        pdf_reader = PyPDF2.PdfFileReader(rendered_report)
+        rendered_text = pdf_reader.getPage(1).extractText()
+
+        self.assertIn('Multiline text question?', rendered_text)
+        self.assertIn('Not answered', rendered_text)
+
+    def test_radiobutton_rendered(self):
+        radio_question = '''[
+        {"answer": 0,
+        "id": 1,
+        "section": 1,
+        "question_text": "A radiobutton question?",
+        "choices": [
+        {"id": 0, "choice_text": "This is radiobutton choice 0"},
+        {"id": 1, "choice_text": "This is radiobutton choice 1"},
+        {"id": 2, "choice_text": "This is radiobutton choice 2"},
+        {"id": 3, "choice_text": "This is radiobutton choice 3"},
+        {"id": 4, "choice_text": "This is radiobutton choice 4"}
+        ],
+        "extra": {
+        "extra_text": "Extra text for choice 0",
+        "answer": "Extra radiobutton answer text"
+        },
+        "type": "RadioButton"
+        }
+        ]'''
+        report = PDFFullReport(self.report, radio_question)
+        output = report.generate_pdf_report(recipient=None, report_id=None)
+        rendered_report = BytesIO(output)
+        pdf_reader = PyPDF2.PdfFileReader(rendered_report)
+
+        rendered_text = pdf_reader.getPage(1).extractText()
+        self.assertIn('A radiobutton question?', rendered_text)
+        for i in range(5):
```

Comment:
any thoughts on verifying that the given answer is marked as checked here? open to changing the output to make checked vs unchecked a little more obvious. same question for checkbox questions